code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -1825,6 +1825,20 @@
 		reagents.add_reagent(CARPPHEROMONES, 3)
 		bitesize = 3
 
+/obj/item/weapon/reagent_containers/food/snacks/crab_sticks
+	name = "Not-Actually-Imitation Crab sticks"
```

Comment:
I'll do it for that one, but all the other snacks should have it. If anyone has nothing else to do.